pyproject.toml

```diff
@@ -49,6 +49,7 @@ distlib = "0.3.9"
 ipython = "8.36.0"
 setuptools = ">65.5.1"
 pre-commit = ">=3.0.0"
+jupyter-sphinx = "^0.5.3"
```

Comment:
@vprusso It needs to be rebased due to this addition in the `toml` file, as well as the updated poetry lock file.

Reply:
Updated in eb511a0d2fc3eb5539e683bbc89c9330a36e47c8

Reply:
It's merged with `master`, so I think this is the behavior we are after if I'm not mistaken.

Reply:
My apologies, I was not paying attention to the diff. Looks like `^` was added.